src/Symfony/Component/Finder/Iterator/ReverseSortingIterator.php

```diff
@@ -1,32 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Finder\Iterator;
-
-/**
- * Reverse the order of a previous iterator.
- *
- * @author Grégoire Pineau <lyrixx@lyrixx.info>
- */
-class ReverseSortingIterator implements \IteratorAggregate
```

Comment:
this class should be deprecated, not removed, as it was not marked internal

Reply:
This class only exists in master (as the PR was merged a few hours ago only).

Reply:
hmm indeed, I missed that